streaming-atomfeed/src/main/resources/hibernate.default.properties

```diff
@@ -16,10 +16,10 @@
 # Default (overridable) with runtime properties file
 
 # Connection Properties -->
-hibernate.connection.username=root
-hibernate.connection.password=test
+hibernate.connection.username=admin
+hibernate.connection.password=BHPmyMtaLj6S
```

Comment:
If these are not standard in OpenMRS, let's revert these (to prevent the impression that there is naything specific with these).

Reply:
@bashir2 should we use a `.templates` files / another approach to more easily allow individual user settings to be kept as a `hibernate.default.properties` file without checking it into github?